tests/integration/components/billing/selected-addon-test.js

```diff
@@ -0,0 +1,35 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import profilePage from 'travis/tests/pages/profile';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+
+module('Integration | Component | selected-addon', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function (assert) {
+
+    this['actions'] = {
+      goToFirstStep: () => { }
+    };
+
+    const addonConfig = {
+      id: 'credits_500k',
+      name: '500 000 credits (50k Linux build minutes)',
+      price: 30000,
+      quantity: 500000,
+      type: 'credit_private'
+    };
+
+    this.selectedAddon = addonConfig;
+    this.set('selectedAddon', addonConfig);
```

Comment:
I don't think this duplication is needed:
```suggestion
    this.set('selectedAddon', addonConfig);
```